languages/sr_RS/Zend_Validate.php

```diff
@@ -0,0 +1,308 @@
+    "The input does not appear to be a valid date" => "Унет је неисправан датум",
+    "The input does not fit the date format '%format%'" => "Унос не одговара формату датума '%format%'",
+
+    // Zend\Validator\DateStep
+    "Invalid type given. String, integer, array or DateTime expected" => "Погрешан тип унет. Очекиван је текст, цео број, низ или датум",
+    "The input does not appear to be a valid date" => "Унет је неисправан датум",
+    "The input does not fit the date format '%format%'" => "Унос не одговара формату датума '%format%'",
+    "The input is not a valid step" => "Унос није исправан корак",
+
+    // Zend\Validator\Db\AbstractDb
+    "No record matching the input was found" => "Није пронађен запис који одговара уносу",
+    "A record matching the input was found" => "Пронађен је запис који одговара уносу",
+
+    // Zend\Validator\Digits
+    "The input must contain only digits" => "Унос мора да садржи само бројеве",
+    "The input is an empty string" => "Унет је празан текст",
+    "Invalid type given. String, integer or float expected" => "Погрешан тип унет. Очекиван је текст, цео број или стварни број",
+
+    // Zend\Validator\EmailAddress
+    "Invalid type given. String expected" => "Погрешан тип унет. Очекиван је текст",
+    "The input is not a valid email address. Use the basic format local-part@hostname" => "Унета је неисправна адреса е-поште. Користите основни формат local-part@hostname",
+    "'%hostname%' is not a valid hostname for the email address" => "'%hostname%' није исправан hostname за адресу е-поште",
+    "'%hostname%' does not appear to have any valid MX or A records for the email address" => "'%hostname%' нема исправних MX или A записа за адресу е-поште",
+    "'%hostname%' is not in a routable network segment. The email address should not be resolved from public network" => "'%hostname%' није у рутабилном мрежном сегменту. Адреса е-поште не би требало да се одређује са јавне мреже",
+    "'%localPart%' can not be matched against dot-atom format" => "'%localPart%' се не може упарити у dot-atom формату",
+    "'%localPart%' can not be matched against quoted-string format" => "'%localPart%' се не може упарити у quoted-string формату",
+    "'%localPart%' is not a valid local part for the email address" => "'%localPart%' није исправан локални део за адресу е-поште",
+    "The input exceeds the allowed length" => "Унос превазилази дозвољену дужину",
+
+    // Zend\Validator\Explode
+    "Invalid type given" => "Погрешан тип унет",
+
+    // Zend\Validator\File\Count
+    "Too many files, maximum '%max%' are allowed but '%count%' are given" => "Превише фајлова, дозвољени максимум је '%max%' а унет је '%count%'",
+    "Too few files, minimum '%min%' are expected but '%count%' are given" => "Премало фајлова, очекивани минимум је '%min%' а унет је '%count%'",
+
+    // Zend\Validator\File\Crc32
+    "File does not match the given crc32 hashes" => "Фајл не одговара датим crc32 вредностима",
```

Comment:
I would not add quotations, it will be to much of them in some cases, and they are already in use for variables in messages.